x/clarifyx.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2022-2025 Searis AS
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clarifyx
+
+import (
+	"github.com/clarify/clarify-go"
+	"github.com/clarify/clarify-go/fields"
+	"github.com/clarify/clarify-go/internal/request"
+	"github.com/clarify/clarify-go/jsonrpc"
+	"github.com/clarify/clarify-go/views"
+)
+
+const (
+	apiVersion             = "1.1"
+	apiVersionExperimental = "1.2alpha1"
+
+	paramFormat      jsonrpc.ParamName = "format"
+	paramIntegration jsonrpc.ParamName = "integration"
+	paramItem        jsonrpc.ParamName = "item"
+	paramQuery       jsonrpc.ParamName = "query"
+)
+
+type Client struct {
+	clarify.Client
+}
+
+func Upgrade(c *clarify.Client) Client {
+	return Client{Client: *c}
+}
+
+func (c Client) Admin() AdminNamespace {
+	return AdminNamespace{AdminNamespace: c.Client.Admin()}
+}
+
+type AdminNamespace struct {
+	clarify.AdminNamespace
+}
+
+// ConnectSignals returns a new request for publishing signals as items.
+func (ns AdminNamespace) ConnectSignals(integration, item string, q fields.ResourceQuery) ConnectSignalsRequest {
+	return methodConnectSignals.NewRequest(ns.Handler(),
```

Comment:
@smyrman Was able to create a `x` package as previously discussed. However, I weren't able to get the private `jsonrpc.Handler` from the original Client. I created a public method to retrieve it. Is this okay, or do you know of any other way?

Reply:
I am OK wit this solution 👍